catalog/service/impl/src/main/java/org/projectnessie/catalog/service/impl/CatalogServiceImpl.java

```diff
@@ -156,7 +156,6 @@ public Stream<Supplier<CompletionStage<SnapshotResponse>>> retrieveSnapshots(
               try {
                 snapshotId = snapshotIdFromContent(c.getContent());
               } catch (Exception e) {
-                e.printStackTrace();
```

Comment:
Should we re-throw `e` here? I know this is beyond the cosmetic cleanup, but since it caught our attention, we might as well fix it, perhaps? Also, I do not think an exception is actually expected in this case... WDYT?

Reply:
But not in this PR

Reply:
Meh - added a logging here